### PR TITLE
User list for starting new conversations

### LINKED ISSUE #49: Add user list for starting new conversation
Adds a searchable user list screen to pick a user to start a new conversation with

## Mockups

image

## Story

#19

## Estimate

6h

### PR DESCRIPTION
**Pull request by bschne**
Implements a searchable user list for starting new chat conversations (using mock data).

Resolves #49﻿

## Screenshots

CleanShot 2021-04-22 at 20 49 16@2x
CleanShot 2021-04-22 at 20 49 22@2x
CleanShot 2021-04-22 at 20 49 28@2x

### CODE REVIEW

**Review by nehasingh1491 — DISMISSED** [review]
Approved

File: src/components/chat/Contacts.js [hunk 1]
```diff
@@ -1,10 +1,117 @@
...
     return (
-      <h1>Show all chat contacts (i.e. users) here</h1>
+      <div className="flex flex-col">
+        <div className="flex-none bg-gray-300 text-center h-12">
+          <h1 className="font-bold text-xl align-middle pt-2.5">Start New Conversation</h1>
+          <div className="absolute -top-1 -left-2 cursor-pointer" onClick={() => this.redirectBackToChat()}>
+            <Back></Back>
+          </div>
+        </div>
+        <div className="flex-non bg-gray-200 p-2">
```
Comment (celine-s) [inline src/components/chat/Contacts.js]: typo? flex-none?

Comment (bschne) [reply src/components/chat/Contacts.js]: Good catch, thx 👍

**Review by celine-s — APPROVED** [review]
nice

**Pull request merged**